pynumaflow/info/info_server.py

```diff
@@ -0,0 +1,59 @@
+import os
+from importlib.metadata import version
+from typing import List, Tuple
+
+from pynumaflow import setup_logging
+from pynumaflow.info.info_types import ServerInfo, EOF
+import json
+import logging
+
+_LOGGER = setup_logging(__name__)
+if os.getenv("PYTHONDEBUG"):
+    _LOGGER.setLevel(logging.DEBUG)
+
+
+def get_sdk_version() -> str:
+    """
+    Return the pynumaflow SDK version
+    """
+    try:
+        return version("pynumaflow")
+    except Exception as e:
+        # Adding this to handle the case for local test/CI where pynumaflow
+        # will not be installed as a package
+        _LOGGER.error(f"Could not read SDK version {e}")
+        return "Could not read SDK version"
```

Comment:
I think in numaflow-java and numaflow-go, if can't read SDK version, we return "". Should we update it to make the contract consistent?

Reply:
Done